card/lib/tasks/card/migrate.rake

```diff
@@ -94,7 +94,7 @@ namespace :card do
     desc "write the version to a file (not usually called directly)"
     task :stamp, :type do |_t, args|
       ENV["SCHEMA"] ||= "#{Cardio.gem_root}/db/schema.rb"
-      Cardio.config.action_mailer.perform_deliveries = false
+      Card.config.action_mailer.perform_deliveries = false
```

Comment:
I don't think Card is already loaded here, so Cardio may make more sense.

Reply:
That's the rule of thumb for deciding which to use.  We want to avoid loading Card to get the config object.